From a0f2eefcae7f321a16e1be318e60b98bc0439af1 Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Wed, 30 Oct 2019 10:30:40 -0500
Subject: [PATCH 047/151] HSD #1707095796: Add Agilex SMMU clocks

commit  720cd5d0c0acc6bc1a24c408aef8ca80eb2465b2 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

arm64: dts: agilex: add SMMU clocks

Add the SMMU clocks needed for Agilex SMMU.

Has dependency on HSD #2209582664:
https://pg-rdcrucible.altera.com:8080/cru/LINUX-SOCFPGA-1535

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 91afba3d72c7..cc0e3ae176ff 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -346,6 +346,9 @@
 				<0 162 4>, <0 163 4>, <0 164 4>, <0 165 4>,
 				<0 166 4>, <0 167 4>, <0 168 4>, <0 169 4>;
 			stream-match-mask = <0x7ff0>;
+			clocks = <&clkmgr AGILEX_MPU_CCU_CLK>,
+				 <&clkmgr AGILEX_L3_MAIN_FREE_CLK>,
+				 <&clkmgr AGILEX_L4_MAIN_CLK>;
 			status = "disabled";
 		};
 
-- 
2.26.1

